views/js/qtiCreator/renderers/Tooltip.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA;
+ */
+/**
+ * @author Christophe Noël <christophe@taotesting.com>
+ */
+define([
+    'lodash',
+    'taoQtiItem/qtiCommonRenderer/renderers/Tooltip',
+    'taoQtiItem/qtiCreator/widgets/static/tooltip/Widget'
+    // 'tpl!taoQtiItem/qtiCreator/tpl/tooltip'
```

Comment:
not needed after all ? (or is it a preparation for next sprint, which would be ok I suppose ?)

Reply:
Yes this is placeholder for next sprint ;-)